### ISSUE TITLE
Requires to put name individually

### ISSUE THREAD

**Issue by ishasingh0172** [arcanis/secretsanta#37]
It's a helpful tool and would be great if it supported csv or list of names for corporate use cases.

**Comment by vuppalarohitanjan** [comment]
Adding to above would kindly add the link in the CSV file, as of now only the name are visible kindly link as well

**Comment by sreehari-ma** [comment]
yes , if the link is also included in the CSV file , this can be useful for mail merge purposes

**Comment by Rickdiculousme** [comment]
This might help a bit for large input cases with exclusions. The output is still a bit manual. I made this Excel helper tool a few years ago and kept improving it so I figured I'd share if it helps others https://github.com/Rickdiculousme/SecretSanta-Excel-Helper
